docs/2_4_common_schema.adoc

```diff
@@ -346,7 +346,7 @@ A value latexmath:[v_{unit}] in `Unit` is converted to the base unit latexmath:[
 
 [latexmath]
 ++++
-v_{base} = \texttt{factor} * v_{unit} + \texttt{offset}
+v_{base} = \texttt{factor} * v_{unit} + \texttt{if relativeQuantity then 0 else offset}
```

Comment:
Should a  similar adaptation not be explicily made for
_(in this case the relativeQuantity of the <TypeDefinition>, see below, has to be taken into account in order to determine whether offset shall or shall not be utilized):_
`factor(v1) * v1 + offset(v1) = factor(v2) * v2 + offset(v2)`
in the paragraph "Unit check when connecting variables of different FMUs"

Shouldn't we explain, that `factor `and `offset ` are properties of the BaseUnit, while `relativeQuantity` is an attribute of a type and link to its definition

Reply:
Yes that would be better

Reply:
@t-sommer : in which order shall we proceed?